Sources/SignalProducer.swift

```diff
@@ -65,6 +65,23 @@ public struct SignalProducer<Value, Error: Swift.Error> {
 		}
 	}
 
+	/// Creates a producer for a `Signal` that immediately sends one value, then
+	/// completes.
+	///
+	/// This initializer differs from `init(value:)` in that it will not 
+	/// evaluate the closure supplying its sole `value` event until the 
+	/// `SignalProducer` is started.
+	///
+	/// - parameters:
+	///   - block: A block that supplies a value to be sent by the `Signal` in
+	///            a `value` event.
+	public init(_ block: @escaping () -> Value) {
```

Comment:
I think the unnamed parameter best fits with the other unnamed lossless-type-converting functions that we already have.

I'd also really like to add this variant:

```
public init<Value, Error>(_ block: @escaping ((Result<Value, Error>) -> Void) -> Void)
```